### PR TITLE
[eclipse/xtext#1897] converted xtend code to java

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1897] converted xtend code to java
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/jvmmodel/JvmTypesBuilderTest.java [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+		});
+		Assert.assertEquals(1, op.eAdapters().size());
+		Assert.assertNull(containerProvider.getLogicalContainer(expr));
+	}
+
+	@Test
+	@Ignore
+	public void testAddNull() throws Exception {
+		BasicEList<String> list = new BasicEList<String>();
+		Assert.assertTrue(list.isEmpty());
```
Comment (miklossy) [inline org.eclipse.xtext.xbase.tests/src/org/eclipse/xtext/xbase/tests/jvmmodel/JvmTypesBuilderTest.java:298]: The following comment is missing that explains why the test case is currently ignored: `//FIXME list += null`

**Comment by miklossy** [comment]
I checked out your branch, executed the test cases on Windows, they all succeed.

The PR looks good, apart from the missing comment (see above).

**Review by miklossy — APPROVED** [review]
